qiita_ware/processing_pipeline.py

```diff
@@ -158,13 +187,68 @@ def _get_preprocess_fastq_cmd(raw_data, prep_template, params):
     forward_seqs = sorted(forward_seqs)
     reverse_seqs = sorted(reverse_seqs)
     barcode_fps = sorted(barcode_fps)
-    mapping_fps = sorted(mapping_fps)
 
-    # Create the split_libraries_fastq.py command
-    cmd = str("split_libraries_fastq.py --store_demultiplexed_fastq -i %s -b "
-              "%s -m %s -o %s %s"
-              % (','.join(forward_seqs), ','.join(barcode_fps),
-                 ','.join(mapping_fps), output_dir, params_str))
+    # Create a temporary directory to store the split libraries output
+    output_dir = mkdtemp(dir=qiita_config.working_dir, prefix='slq_out')
```

Comment:
I noticed that it is really hard to check if a given instance of split libraries is running by checking the file system because the prefix is not informative enough. Can you change to `prefix='slq_out_%d_%d_' % (prep_template.id, raw_data.id)` ?